Assignment1.java

```diff
@@ -0,0 +1,24 @@
+package Zemoso1;
+
+public class Assignment1 {
```

Comment:
Follow naming conventions for naming a class 
Instead of Assignment1 keep some meaningful name which describes what the code is for

Your PR description should describe what the PR is for